posts/2022-06-06-introduction-cypress-en.adoc

```diff
@@ -0,0 +1,217 @@
+Cypress focuses on end-to-end testing (E2E).
+The E2E tests allow us to validate the user experience through predefined scenarios. 
+These scenarios will describe a group of actions that an average user will perform on our site. 
+These tests do not care about the internal implementation of our product, but will adopt the point of view of a user to verify the general behaviour of our product.
+This javascript dependency in your front-end projects will allow you to test your website from a user point of view.
+
+++++
+<div style="width:100%;height:0;padding-bottom:56%;position:relative;"><iframe src="https://giphy.com/embed/l3q2Uzjt0QB4oL7l6" width="100%" height="100%" style="position:absolute" frameBorder="0" class="giphy-embed" allowFullScreen></iframe></div><p><a href="https://giphy.com/gifs/jerseydemic-l3q2Uzjt0QB4oL7l6">via GIPHY</a></p>
+++++
+
+Let's move on to a practical case study to see how to use Cypress and hopefully make you want to try it too!
+
+== II/ Case study
+=== 1. Preparation
+
+For this case study, we will use the Lunatech blog: https://blog.lunatech.com/. 
+We will implement the following scenario:
+
+. Access to the blog
+. Check that there is at least one article on the home page
+. Filter by author X
+. Check that all remaining articles are made by author X
+
+The first thing to do is to open our JS/TS project and https://docs.cypress.io/guides/getting-started/installing-cypress[install Cypress].
+
+It's as simple as that:
+[source, Shell]
+----
+yarn add cypress --dev
+OR
+npm install cypress --save-dev
+----
+
+For the example, we will put all our logic in the `cypress` folder and test with the `chrome` browser.
+To save you time and not type the full path to the Cypress executable, I suggest you put this script in your `package.json` :
+[source, Shell]
+----
+  "scripts": {
+    "cypress:open": "cypress open"
+  }
+----
+
+[quote, How to organize the code?]
+In this first article about Cypress, we will organize the code very simply, but in the following article we will explore good practices to have a clean and maintainable code over time.
+
+You can now run:
+[source, shell]
+----
+npm run cypress:open
+----
+
+This command will open a graphical interface to interact with Cypress.
+You should see the screen below:
+
+image::first-screen-cypress.png[Home page, width = 700]
+
+On this screen, you can choose *E2E testing* (we will come back to all the subtleties of Cypress in the next article including *Component Testing*).
+
+image::browser-cypress.png[Browser selection, width = 700]
+
+You can choose different browsers. 
+In our case, we will use _Chrome_.
+Cypress will then create a base project to start your tests.
+If all went well, you should have this file structure:
+
+image::fichiers-cypress.png[Files created by Cypress, width = 300]
+
+Now you have everything in place to move on to the practical case. 
+Don't close the GUI so you can see your changes in real time!
+
+=== 2. Realization of the practical case
```

Comment:
```suggestion
=== 2. Implementation
```
This is the simplest way to express what i _think_ you mean here - if you'd prefer something less succinct then, perhaps:

```suggestion
=== 2. Building out the example
```